gc/gc-iceberg-files/build.gradle.kts

```diff
@@ -78,9 +78,9 @@ dependencies {
 
 tasks.withType(Test::class.java).configureEach {
   systemProperty("aws.region", "us-east-1")
+  val tmpdir = project.buildDir.resolve("tmpdir")
```

Comment:
Moves the reference to `project` out of _task execution_, which is mandatory to enable Gradle's configuration cache.